tests/search_test.cpp

```diff
@@ -14,8 +14,8 @@ bool test_search() {
   SimpleCP *ss = new SimpleCP(total_time);
 
   Workload wl(100, 1, total_time);
-  wl.add_spec(TimedSpec(Comp(Indiv(metric_t::CENQ, 0), op_t::LE, (uint)1),
-                           time_range_t(last_t, last_t), total_time));
+  Comp* comp_spec = new Comp(Indiv(metric_t::CENQ, 0), op_t::LE, static_cast<uint>(1));
```

Comment:
Is there any reason for using `<static_cast>`? I prefer () cast, it's more compact.

Reply:
[Remove unnecessary static cast](https://github.com/all-things-networking/fperf/pull/27/commits/569385d6503d7aa41026cb9f301ea0f303266bb0)